trezor/trezor-suite
# feat(navigation): Updated Settings menu and Tor

**About**
Revamped settings and Tor menu (top right corner of the app)

**How to test**
Please check linked tasks:
- resolves #3712
- resolves #3723
- slightly refactored Dropdown component (please test Dropdown instances app-wide and on Suite landing page)

TODOs for another PR:
- allow Dropdown component to open menu on hover instead of click (requires deeper refactoring)
- Tor address in global state (?)
